spec/requests/worldpay_missed_payment_forms_spec.rb

```diff
@@ -74,20 +72,26 @@
         sign_in(user)
       end
 
-      it "redirects to the transient_registration page" do
-        post "/bo/resources/#{transient_registration._id}/payments/worldpay-missed", worldpay_missed_payment_form: params
-        expect(response).to redirect_to(resource_finance_details_path(transient_registration._id))
-      end
-
-      it "creates a new payment" do
+      it "redirects to the transient_registration page, creates a new payment and assigns the correct updated_by_user to the payment" do
```

Comment:
This is more about being consistent with the other change I've suggested (which I think is needed because the current description is incorrect)

```suggestion
      it "redirects to the transient_registration's finance details page, creates a new payment and assigns the correct updated_by_user to the payment" do
```